src/main/java/core/basesyntax/interfaces/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface AreaCalculator {
+    double getArea();
```

Comment:
Actually we are not getting area, we calculating it, get if ofted associated with getting some field/value e.t.c. Maybe it makes sence to rename method